ProcessesApi.Tests/V1/E2ETests/Steps/UpdateSoleToJointProcessSteps.cs

```diff
@@ -237,5 +244,17 @@ public async Task ThenTheProcessUpdatedEventIsRaisedWithAppointmentDetails(ISnsF
             };
             await VerifyProcessUpdatedEventIsRaised(snsFixture, processId, oldState, newState, verifyData).ConfigureAwait(false);
         }
+
+        public async Task ThenTheProcessClosedEventIsRaisedWithReason(ISnsFixture snsFixture, Guid processId, string newState)
```

Comment:
I don't think you need `newState` as a param because it will only ever go to ProcessClosed.